src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1215,6 +1215,15 @@ TIP: Since it runs a graphical application, the task will never exit and run for
 you want to check manually some results via the graphical application and then exit the application.
 The task will then terminate normally.
 
+=== Troubleshoot a Workflow Execution
+
+When a task has *IN_ERROR* status, a right click on it allows you to select _Mark as finished and resume_. 
+This will set the task status as *FINISHED* and then resume job execution. 
+This allows an operator to manually perform actions to the *IN_ERROR* task, without cancelling the whole job execution.
+
+If the task has not *IN_ERROR* status, a right click on it shows the _Mark as finished and resume_ option, but it appears in grey: the action is unavailable.
+
```

Comment:
how about this

If a task is not in *IN_ERROR* status, the `finished and resume` option will be in grey, being said the action is disabled.